doc/compiler/jitserver/Usage.md

```diff
@@ -20,143 +20,243 @@ OpenJDK Assembly Exception [2].
+$ java -XX:JITServerAddress=example.com
 
-JITServer is currently a technology preview. Its use is not yet supported
-#JITServer: JITServer version: 1.17.0
-#JITServer: JITServer Client Mode. Server address: localhost port: 38400. Connection Timeout 2000ms
-#JITServer: Identifier for current client JVM: 9238414254742342824
-...
-```
+-or-
 
-### JITServer heartbeat
-Server has a statistics thread running in the background, which can periodically print general information
-about the state of the server. Use `-Xjit:statisticsFrequency=<period-in-ms>` option to enable heartbeat logging.
-The output looks like this:
+$ java -XX:JITServerAddress=127.0.0.1
 ```
-$ jitserver -Xjit:statisticsFrequency=3000` # print heartbeat info every 3 seconds
-
-JITServer is currently a technology preview. Its use is not yet supported.
 
-JITServer is ready to accept incoming requests
-#JITServer: Number of clients : 0
-#JITServer: Total compilation threads : 63
-#JITServer: Active compilation threads : 1
-#JITServer: Physical memory available: 12570 MB
-#JITServer: CpuLoad 6% (AvgUsage 1%) JvmCpu 6%
-```
+By default, the server address is set to `localhost`, i.e. server and client are on the same machine.
 
-## Configuration
+### Port
 
-### Hostname
-On the client, you can specify the server hostname or IP.
-```
-$ java -XX:JITServerAddress=example.com
-```
+By default, communication occurs on port `38400`. You can change this by specifying the `-XX:JITServerPort` suboption as follows:
 
-### Port
-By default, communication occurs on port 38400. You can change this by specifying the `-XX:JITServerPort` suboption as follows:
 ```
 $ jitserver -XX:JITServerPort=1234
 $ java -XX:+UseJITServer -XX:JITServerPort=1234 MyApplication
 ```
 
 ### Timeout
+
 If your network connection is flaky, you may want to adjust the timeout. Timeout is given in milliseconds using `-XX:JITServerTimeout` suboption. Client and server timeouts do not need to match. By default there is timeout of 30000 ms at the server and 2000 ms at the client. Typically the timeout at the server can be larger; it can afford to wait because there is nothing else to do anyway. Waiting too much at the client can be detrimental because the client has the option of compiling locally and make progress.
+
 ```
 $ jitserver -XX:JITServerTimeout=5000
 $ java -XX:+UseJITServer -XX:JITServerTimeout=5000 MyApplication
 ```
 
 ### Encryption (TLS)
-By default, communication is not encrypted. If messages sent between the client and server need to traverse some untrusted network, you may want to set up encryption. Encryption reduces performance, so consider whether it is required for your use case.
+
+By default, communication is not encrypted. If messages sent between the client and the server need to traverse some untrusted network, you may want to set up encryption. Encryption reduces performance, i.e. compilations take longer, so consider whether it is required for your use case.
```

Comment:
communication --> network communication

"compilations take longer" --> "compilations take longer and consume more CPU"